Documentation/ApiOverview/TypoScriptSyntax/Syntax/Conditions.rst

```diff
@@ -298,3 +306,122 @@ Summary
 - Conditions can be used outside of confinements (curly braces) only.
   However the :code:`[GLOBAL]` condition will always break a confinement if
   entered inside of one.
+
+
+.. _typoscript-syntax-conditions-migration-sel:
+
+Migration from the pre-Symfony condition syntax
+===============================================
+
+.. versionchanged:: 9.4
+
+   The old condition syntax got deprecated in TYPO3 version 9.4 and removed in 10
+   with the introduction of this change
+   :doc:`t3core:Changelog/9.4/Feature-85829-ImplementSymfonyExpressionLanguageForTypoScriptConditions`
+
+
+For information on how to migrate from the old pre-SEL style TypoScript conditions
+to the new SEL-conditions please see below:
+
+
+Variables and functions
+-----------------------
+
+For a detailed list of the available objects and functions refer to the
+:ref:`the TypoScript Reference <t3tsref:conditions>`.
+
+Variables::
+
+   [page["backend_layout"] == 1]
+      # Old syntax: [globalVar = TSFE:page|backend_layout = 1]
```

Comment:
Here'd suggest using a `diff` code block.

Reply:
I think we can leave it like that, this is working TypoScript with just a comment ppl can copy it like that and use it.